README.md

```diff
@@ -499,6 +499,70 @@ lvm_volume_group 'vg00' do
 end
 ```
 
+### lvm_thin_pool_meta_data
+
+Manages LVM thin pool metadata size.
+
+#### Actions
+
+Action  | Description
+------- | ---------------------------------------------------------------
+:resize | Resize an existing thin pool metadata volume
+
+#### Properties
+
+<table>
+  <tr>
+    <th>Attribute</th>
```

Comment:
If we can do this with a normal markdown table we should. Sometime we really need to convert the other resources to markdown as well. Supermarket displays it better that way.